https://bugs.gentoo.org/767796

From: Robert-André Mauchin <zebob.m@gmail.com>
Date: Mon, 14 Dec 2020 02:23:52 +0000 (+0100)
Subject: hls: add missing <limits> include
X-Git-Url: https://git.videolan.org/?p=vlc.git;a=commitdiff_plain;h=3aea33378b0b317b99fce2eeaca8b31bf7cc6aba

hls: add missing <limits> include

Compilation would fail with:

> error: 'numeric_limits' is not a member of 'std'

without this include because of some hardening build
flags mandated on Fedora Linux, with GCC 11.

Refs #25325

Signed-off-by: Robert-André Mauchin <zebob.m@gmail.com>
Signed-off-by: Alexandre Janniaux <ajanni@videolabs.io>
---

--- a/modules/demux/hls/playlist/Parser.cpp
+++ b/modules/demux/hls/playlist/Parser.cpp
@@ -42,6 +42,7 @@
 #include <map>
 #include <cctype>
 #include <algorithm>
+#include <limits>
 
 using namespace adaptive;
 using namespace adaptive::playlist;
